ee/packages/git-sync-manager/src/pull-request/pull-request.controller.ts

```diff
@@ -85,10 +85,11 @@ export class PullRequestController {
     });
 
     try {
-      const pullRequest = await KafkaPacemaker.wrapLongRunningMethod<string>(
-        context,
-        () => this.pullRequestService.createPullRequest(validArgs)
-      );
+      const { pullRequestUrl, diffStat } =
+        await KafkaPacemaker.wrapLongRunningMethod<{
+          pullRequestUrl: string;
+          diffStat: any;
```

Comment:
If diffStat is actually a string, why use `any`?

Reply:
probably a leftover. It should be a string here